release.md

```diff
@@ -0,0 +1,28 @@
+# falcoctl Release Process
+
+Our release process is automated using [goreleaser](https://github.com/goreleaser/goreleaser).
+
+When we release we do the following process:
+
+1. We decide together (usually in the #falco channel in Sysdig's slack) what's the next version to tag
+2. A person with repository rights does the tag
+3. The same person runs goreleaser in their machine following the "Release commands" section below
+4. The tag is live on Github with the artifacts
+5. Travis builds the tag and push the related docker images
+
+## Release commands
+
+Tag the version
+
+```bash
+git tag -a v0.1.0-rc.0 -m "v0.1.0-rc.0"
+git push origin v0.1.0-rc.0
+```
+
+Run goreleaser, make sure to export your GitHub token first.
+
+```
+export GITHUB_TOKEN=<YOUR_GH_TOKEN>
+make release
```

Comment:
where's Makefile?

Reply:
I didn't do the makefile in the end! Updating this! thanks